security/openpgp_message.ksy

```diff
@@ -448,26 +448,26 @@ enums:
     110: private_experimental_algorithm_10
 
   subpacket_types:
-    0: reserved
-    1: reserved
+    0: reserved_0
+    1: reserved_1
```

Comment:
Should be rather `reserved0`, `reserved1` per the [style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id):

> For `reserved` fields which are known to be unused, use reserved name (or `reserved1`, `reserved2`, etc, if there are many of them)

Although this is not a field but an enum key, there is no reason to use a different convention.

Reply:
Alrighty then. I also fixed a few other instances of that.

Reply:
For enums it is proposed to use a different convention - to use enum number value instead of sequence numbers. But here they match.